scripts/User_Management/grade_9_mode.py

```diff
@@ -0,0 +1,41 @@
+from scripts._utils import ssh, pi, utils
+
+host_themes = "pi-themes"
+tvs = ["1", "2", "3", "4"]
```

Comment:
Don't redefine/duplicate code.  keep it dry and import these from the source.

Reply:
im confused